src/gnatcoll-email-utils.adb

```diff
@@ -1695,27 +1685,14 @@ package body GNATCOLL.Email.Utils is
 
       if Set = Charset_US_ASCII then
          Encoding := Encoding_7bit;
-      elsif Set = Charset_ISO_8859_1
```

Comment:
How would you want this code to be rewritten?
given that you want ```remove code reformatting, and preserve formatting logic in sections that are being refactored.```

Note that the newly rewritten code must be pretty printed,
since otherwise the rewritten code might violate style rules (as specified in the project file).

Reply:
In this case a too long line is certainly possible ;-)

Reply:
I am referring to places where reformatting is not motivated by anything, like gnatcoll-memory.adb:291

Reply:
> How would you want this code to be rewritten?
In this specific case, try to keep the same terms on each line as it was previously. We used to have one line for latin-3 and another for latin-4, your patch reformats it so references to latin-4 are on two different lines, this harms readability.

Reply:
You did however not turn off pretty printing, 
so running the pretty printer might have changed your constraints.

It is an automated change, but it is a nice case of requirements for rewriting existing code.

Reply:
> You did however not turn off pretty printing,

I don't understand. "turn off pretty printing" where? gnatcoll is obviously hand-formatted.